release-notes/unreleased/2102.other.rst

```diff
@@ -0,0 +1,2 @@
+IBM Cloud accounts will now use an access token to call the Qiskit Runtime API instead of the 
+token provided by the user. 
```

Comment:
From the user perspective, what is the difference between the previous state and the new state? Asking for my own understanding. It feels like users don't need to change the provided token, right?

Reply:
From the user perspective there should not be any difference. They will still use their cloud api key for initializing their account. They don't need to change their provided key. 

Even though this change shouldn't effect the user, I think it'd still be good to add a release note.